macros/edr/data_monitoring/anomaly_detection/get_anomaly_scores_query.sql

```diff
@@ -31,6 +31,8 @@
     {%- set detection_end = elementary.get_detection_end(test_configuration.detection_delay) %}
     {%- set min_bucket_start_expr = elementary.get_trunc_min_bucket_start_expr(detection_end, metric_properties, test_configuration.days_back) %}
 
+    {%- set metric_time_bucket_expr = 'case when bucket_start is not null then bucket_start else bucket_end end' %}
```

Comment:
I added this because I think that for timestamped tests it's better to use `bucket_start`, but for non-timestamped tests we currently only fill `bucket_end`
(This is specifically for the interface of `anomaly_exclude_metrics` as explained below)